packages/kbn-esql-validation-autocomplete/src/definitions/functions.ts

```diff
@@ -1973,988 +4537,832 @@ const mvDedupeDefinition: FunctionDefinition = {
           type: 'version',
           optional: false,
         },
+        {
+          name: 'order',
+          type: 'keyword',
+          optional: true,
```

Comment:
@drewdaemon I think this is part of the generating definition script below is causing some definitions to miss this 'asc', 'desc' literal options.
```
  mv_sort: {
    signatures: new Array(6).fill({
      params: [{}, { literalOptions: ['asc', 'desc'] }],
    }),
  },
```

Reply:
Due to us expanding the signatures, no new Array(6) doesn't work anymore. Out of scope for this PR but I think we should update the script to not depend on a static value.

Reply:
If you have ideas to improve the script, go ahead! 

My one ask is that we don't make it too magical/smart. Also, I'm hoping that Elasticsearch will eventually provide us with these "accepted options" automatically so we don't have to fill them in on our side 🤞